data/json/mutations.json

```diff
@@ -1172,7 +1172,17 @@
         "cancels" : ["BEAK", "BEAK_HUM", "BEAK_PECK", "FANGS"],
         "prereqs" : ["MUZZLE_RAT"],
         "threshreq" : ["THRESH_RAT"],
-        "category" : ["MUTCAT_RAT"]
+        "category" : ["MUTCAT_RAT"],
+        "attacks" : {
+            "attack_text_u" : "You bite into %s with your ratlike incisors",
+            "attack_text_npc" : "%1$s bites %2$s with his ratlike incisors",
```

Comment:
A bunch of the NPC attack texts use "his" instead of "their". If you're going to use the singular they instead of checking NPC gender, please be consistent about it.